src/ws-tiles-chart/tile.js

```diff
@@ -49,9 +51,11 @@ export class Tile extends Component {
 
     return (
       <div
-        className={`tile ${this.props.groupName}`}
+        className={`tile ${groupName} ${className}`}
         style={style}
-        onClick={() => this.props.onClick(this.props.groupName, this.props.identifier)}
+        onClick={() => this.props.onClick(groupName, identifier)}
```

Comment:
The tile chart contains multiple groups that can be shown in different colors (depending on your css) and each group contains multiple tiles.
- groupName identifies the group of the tile that was clicked 
- identifier identifies a single tile